automated install/basic-install.sh

```diff
@@ -319,53 +320,57 @@ It is also possible to use a DHCP reservation, but if you are going to do that,
 setDHCPCD() {
 	# Append these lines to dhcpcd.conf to enable a static IP
 	echo "## interface $piholeInterface
-	static ip_address=$IPv4addr
+	static ip_address=$IPv4_address
 	static routers=$IPv4gw
 	static domain_name_servers=$IPv4gw" | tee -a /etc/dhcpcd.conf >/dev/null
 }
 
 setStaticIPv4() {
+  local IFCFG_FILE
+  local IPADDR
+  local CIDR
 	if [[ -f /etc/dhcpcd.conf ]];then
```

Comment:
This will always exist. We are installing `dhcpcd5` as an installer dependency, and that file is automatically generated. So every Debian and derivative will have this file no matter what based on current install. Need ideas on how to handle this situation. Take `dhcpcd5` out of the install dependencies, but then how should we proceed?